core/storage/collection/gae_models_test.py

```diff
@@ -500,11 +500,10 @@ def test_get_all_non_private_commits(self):
         private_commit.put()
         public_commit.update_timestamps()
         public_commit.put()
-        commits = (
+        results, _, _ = (
```

Comment:
See previous notes on `more`, here and below.

Reply:
Done.